algoliasearch/src/main/java/com/algolia/search/saas/BaseIndex.java

```diff
@@ -42,6 +44,9 @@
     private APIClient client;
     private String encodedIndexName;
     private String indexName;
+    private ExpiringCache<String, String> searchCache;
```

Comment:
As we discussed, it would be better to store a raw `byte[]` (UTF-8) instead of a `String` (UTF-16), as that would be more compact in most use cases.